Nest/00. Plan.md

```diff
@@ -0,0 +1,53 @@
+# 1 Introduction
+## 1.1 What is a package manager?
+
+## 1.2 Packages and dependencies
+
+## 1.3 Repositories
+
+## 1.4 Package's syntax
+
+# 2 Nest
+## 2.1 Nest's options
+
+## 2.2 Nest commands
+### 2.2.1 Updating the list of available packages
+
+### 2.2.2 Installing and uninstalling a package
+
+### 2.2.3 Updating the system
+
+## 2.3 Summary
+
+# 3 Advanced concepts
+## 3.1 Managing repositories
+### 3.1.1 The official repositories
+### 3.1.2 The unofficial repositories
+### 3.1.3 Configuring repositories
+
+## 3.2 Overview of the cache
+
+## 3.3 Version requirement
+
+# 4 Finest
+## 4.1 Key concepts
+### 4.1.1 The cache in detail
+### 4.1.2 The transactions
+### 4.1.3 The dependency graph
+### 4.1.4 Groups
+### 4.1.5 Example of dependency graph transactions
+
+## 4.2 Finest's options
+
+## 4.3 Finest's commands
+### 4.3.1 Managing packages through requirements
+
+### 4.3.2 Updating the system from the dependency graph
```

Comment:
```suggestion
### 4.3.2 Updating the system through the dependency graph
```
Unless I'm understanding the intent wrong. "from" sounds weird.

Reply:
When you update the system with Finest, you're merging a copy of your dependency graph (with all the transactions you performed) with the graph the system is currently using. So I guess in a way, both are correct :thinking: 
Although I think that `through` sounds better

Reply:
So from those comments [there](https://github.com/raven-os/getting-started/pull/1#discussion_r280994531), the title of this part will be `Updating the system by merging the dependency graph`, which solves that problem since there's no `from` nor `through` :p